packages/terra-clinical-site/src/Index.jsx

```diff
@@ -7,7 +7,6 @@ import Home from './Home';
 import styles from './site.scss';
```

Comment:
Technically this file is dead-code and needs to be removed for anyone curious. Terra-dev-site automatically generates the routes from the `generatedComponentConfig.js` file when the site is started.